generators/client/templates/angular/src/main/webapp/app/blocks/config/_auth.config.ts

```diff
@@ -0,0 +1,34 @@
+<%#
+Copyright 2013-2017 the original author or authors from the JHipster project.
+
+This file is part of the JHipster project, see http://www.jhipster.tech/
+for more information.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-%>
+import { AuthConfig } from 'angular-oauth2-oidc';
+
+export const authConfig: AuthConfig = {
+
+    // Url of the Identity Provider
+    issuer: 'http://localhost:9080/auth/realms/jhipster',
+
+    // URL of the SPA to redirect the user to after login
+    redirectUri: window.location.origin + '/index.html',
```

Comment:
Can we make this dynamic? If the user hits a URL that requires authentication, they should be redirected back to it after logging in.